demo-config1.yml

```diff
@@ -0,0 +1,13 @@
+# working with Pac4jFactory1:
+pac4j:
+  configClass: org.pac4j.demo.dw.factory.DemoConfigFactory1
```

Comment:
Remember that the `Pac4jFactoryX` are just proof of concepts, I just wanted to be sure that it was working, but indeed, it must be properly completed with documentation and so on.

Yes, the `callbackUrl` is missing in the YAML file as I set it in the configuration class, just to prove everything can be done at this level.
That said, I think the global idea is to keep as many properties as possible in the YAML file, which is more explicit.
Yes, we'll add comments in the YAML file, but above all, the documentation must be upgraded with the new incoming capabilities (README).